### DISCUSSION TITLE
How to draw colored emoji through Glyphs?

### DISCUSSION THREAD

**Discussion by Valkirie** [wherewhere/ModernWpf#25] [Q&A]
Hey, I'm trying to draw colored emojis using Segoe UI Emoji like the following 🏠🎉 using glyphs but am failing. Can you help me here?

**Comment by wherewhere** [comment]
Try using Textblock?

**Reply by Valkirie** [reply]
Any way to get it displayed like a fonticon, without having to use a textblock?

**Reply by wherewhere** [reply]
But FontIcon is based on TextBlock...
